SalesforceCommerceCloud/pwa-kit
# Upgrade Chakra UI to latest minor version `1.7.1`

# Description



This PR upgrades Chakra to the latest minor version `1.7.1`.

We recently upgraded in this PR: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/181 to Chakra version `1.6.2`.

Chakra version`1.6.2` introduced a "dark mode" bug, enabling "Dark mode" by default instead of "Light mode": https://github.com/chakra-ui/chakra-ui/issues/4987


The issue seems to be fixed in the latest Chakra minor release`1.7.1`: https://chakra-ui.com/changelog#12-11-2021
We don’t seem to be affected by the breaking changes introduced on `1.7`: https://chakra-ui.com/changelog#09-11-2021



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Upgraded Chakra UI to the latest minor version.

# How to Test-Drive This PR

- Checkout this branch `upgrade-chakra`.
- Enable "Dark mode" in your system (macOS  > System Preferences, click General, then select the Dark Appearance)
- Run `npm run start --prefix packages/pwa` and verify that the background of the navigation is white instead of the dark background that we see deployed on `staging` https://scaffold-pwa-staging.mobify-storefront.com/

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
